.circleci/config.yml

```diff
@@ -22,11 +22,11 @@ jobs:
           sudo tar -C /usr/local -xzf go$GOVERSION.$OS-$ARCH.tar.gz
       - run: go version
       # install latest Docker
-      #- run: |
-          #sudo rm -rf /var/cache/apt/archives && sudo ln -s ~/.apt-cache /var/cache/apt/archives && mkdir -p ~/.apt-cache/partial
-          #sudo service docker stop
-          #curl -fsSL https://get.docker.com/ | sudo sh
-          #docker version
+      - run: |
+          sudo rm -rf /var/cache/apt/archives && sudo ln -s ~/.apt-cache /var/cache/apt/archives && mkdir -p ~/.apt-cache/partial
+          sudo service docker stop
+          curl -fsSL https://get.docker.com/ | sudo sh
+          docker version
```

Comment:
had to enable this, 17.06 (old) docker fails and hangs when removing containers that are frozen.

Reply:
I agree, is it ok if I get that in with separate PR?

Reply:
that's fine. if the breakage is mysterious it may be better to add here so that we don't have to run slack support, most users don't update their docker very frequently (usually when something blows up and we tell them to)